src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -18274,8 +18274,8 @@ class Derived : Base<string>
     public override void Test(string s, ref int x) { }
 }
 ";
-            var comp = DiagnosticsUtils.VerifyErrorsAndGetCompilationWithMscorlib(text,
-                new ErrorDescription { Code = (int)ErrorCode.WRN_MultipleRuntimeOverrideMatches, Line = 8, Column = 25, IsWarning = true });
```

Comment:
>WRN_MultipleRuntimeOverrideMatches [](start = 61, length = 34)

Same comment as for the other tests affected by the removal of WRN_MultipleRuntimeOverrideMatches

Reply:
That is tested in `CodeGenOverridingAndHiding`.

---
In reply to: [442522720](https://github.com/dotnet/roslyn/pull/44025#discussion_r442522720) [](ancestors = 442522720)